src/main/java/org/mastodon/mamut/clustering/ClusterRootNodesController.java

```diff
@@ -127,9 +131,21 @@ public void createTagSet()
 		}
 	}
 
-	private void runClassification()
+	private void runClassification( final boolean showDendrogram, final String tagSetName )
 	{
-		List< BranchSpotTree > roots = getRoots();
+		TagSetStructure.TagSet tagSet = null;
+		if ( showDendrogram && tagSetName != null && !tagSetName.isEmpty() )
```

Comment:
Why does this depend on showDendrogram. Does the result change anyhow if this is simplified to 
`if ( tagSetName != null && !tagSetName.isEmpty() )`

Reply:
Yes, indeed. This can be simplified as suggested. I included the `showDendrogram` in the condition, since currently the tagset values are only used, when the dendrogram is shown. However, this may change in the future.